ros2param/ros2param/api/__init__.py

```diff
@@ -47,7 +47,8 @@ def load_parameter_file(*, node, node_name, parameter_file, use_wildcard):
         raise RuntimeError('Wait for service timed out waiting for '
                            f'parameter services for node {node_name}')
     future = client.load_parameter_file(parameter_file, use_wildcard)
-    parameters = list(parameter_dict_from_yaml_file(parameter_file, use_wildcard).values())
+    parameters = list(parameter_dict_from_yaml_file(
+        parameter_file, use_wildcard, target_nodes=[node_name]).values())
```

Comment:
`target_nodes` should be specified with command line arguments, so that it can know what needs to be parsed from the parameter yaml file.

Reply:
It makes sense.